hieradata/org/lsst.yaml

```diff
@@ -1,4 +1,7 @@
 ---
+classes:
+  - "profile::core::icinga_agent"
```

Comment:
Do not put classes at this level in the hierarchy. This will result in the class being included on every node without a way to exclude it.  Please put the inclusion into `profile::core::common` as I suggested.

Reply:
done as suggested